Angular update 20

steinwinde opened the pull request:
This is a first attempt to bring the dev branch from Angular 19 to 20. All official chores to accomplish this have been done and "ng serve" starts without error (but with lots of warnings).

FIXES: WEB-350

JaySoni1 committed:
WEB-117 Loan Account Charge(Amount% charges) Doubling on Application Modification

gkbishnoi07 committed:
WEB-341 Add Integration Tests for CreateClientComponent

steinwinde committed:
Updated Angular 19 to Angular 20

FIXES: WEB-350

LeonMendiolaYV committed:
WEB-347:No Interest Compoundig Option for Deposit,FD and RD accounts (#2710)

Co-authored-by: Yessica Viridiana Leon Mendiola <LeonMendiolaYV>

alberto-art3ch reviewed (changes requested):
Too many Commits in the PR and huge amount of changes in the HTML, high risk to break the UI

steinwinde committed:
Replaces a dependecy in version.js (#2712)

version.js had a dependency on fs-extra, which is
not necessary, because we can use the built-in
Node functionality.

FIXES: WEB-351

steinwinde committed:
Merge pull request #2700 from gkbishnoi07/WEB-341-add-integration-tests-for-create-client-component

WEB-341 Add Integration Tests for CreateClientComponent

alberto-art3ch committed:
Merge pull request #2690 from JaySoni1/WEB-117-loan-account-charge-amount-charges-doubling-on-application-modification

WEB-117 Loan Account Charge(Amount% charges) Doubling on Application Modification

JaySoni1 committed:
WEB-185 Made on date and checked on date should show full datetime with timezone information or explicitly convert the epoch time to UTC!

gkbishnoi07 committed:
Merge pull request #2715 from JaySoni1/WEB-185-made-on-date-and-checked-on-date-should-show-full-datetime-with-timezone-information-or-explicitly-convert-the-epoch-time-to-utc

WEB-185-made-on-date-and-checked-on-date-should-show-full-datetime-with-timezone-information-or-explicitly-convert-the-epoch-time-to-utc

steinwinde committed:
Merged dev into branch.

steinwinde committed:
Fixed a few merge artifacts.

steinwinde commented:
Rather than fixing this PR, I'm providing a new one
